### ISSUE TITLE
[BUG] Failed to delete mongodb data.

### ISSUE THREAD

**Issue by muhammadyusuf-kurbonov** [cweijan/vscode-mysql#695]
- OS: Fedora 36 (Linux kernel: 5.19.15
- Database Type: MongoDB
- Extension Version: 5.8.9
```
SQL:  db('appDatabase').collection("users").deleteMany({_id:{$in:[ObjectID('634dc0433c781c07fad3bfb8'),ObjectID('634dbb450c5175076b080bd8'),ObjectID('634db9cd9eee3303b4325a3b'),ObjectID('634db7bc81b2e9038d7ff20f')]}})

ObjectID is not defined
```
I can't delete items. Neither by one or batch of them.

**Comment by cweijan** [comment]
Thanks for your feedback, I will fix it in the next version.

**Comment by cweijan** [comment]
This bug has been fixed in version 5.8.10.
